.dart_tool/build/entrypoint/build.dart

```diff
@@ -0,0 +1,29 @@
+// ignore_for_file: directives_ordering
```

Comment:
This appears to be an automatically generated/build related file and should not be included in any commits.